docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -186,3 +186,33 @@ List of comma- or space-separated roles that will be mapped to the Admin role.
 ### role_values_grafana_admin
 
 List of comma- or space-separated roles that will be mapped to the Grafana Admin (Super Admin) role.
+
+## [keystore.vault]
+
+### url
+
+Location of the Vault server.
+
+### namespace
+
+Vault's namespace if using Vault with multi-tenancy.
+
+### auth_method
+
+Method for authenticating towards Vault. Vault is inactive if this option is not set. Current possible values: `token`.
+
+### token
+
+ Secret token to connect to Vault when auth_method is `token`.
+
+ ### lease_renewal_interval
```

Comment:
Looks like this h3 and the next two have a space in front, invalidating the heading. Please correct.